frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -35,9 +35,19 @@ i.e.
 
 // Typography
 // prevent orphaned words in short blocks of text
-.text-balance {
+.text-balance,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
   text-wrap: balance;
 }
+// no USWDS setting; can't pass `.measure-none` to Truss component
+.usa-alert__text {
+  max-width: none;
+}
```

Comment:
Global styles add a max-width to paragraphs, causing the alert text to wrap to multiple lines even when there's plenty of room. This custom style feels crude, but we'll revisit when we update our use of alerts. cc @michellemin-nava